### PR TITLE
IEEE-57: Update hackathon information on landing page

### PR DESCRIPTION
**Pull request by Leo6Leo**
## Overview

- Resolves #IEEE-57
We need to update the following things on `landing.html`:

Description

Schedule

FAQs

Use the following FAQ questions:

What is a hackathon?

Keep from last year

Who can apply and what do I do if I have no experience?

Keep from last year

How much does it cost?

Keep from last year

Is this hackathon online?

Yes, this hackathon will be hosted completely online using multiple online communication platforms. So you can enjoy the hacking experience from the comfort of your home.

What software/hardware can we use?

As this hackathon is online, you can use any of the software or hardware that you have available at your house. 

Can we buy hardware and use it in our project?

Yes, you may buy hardware and use it towards your project. However, please remember that this is only a 24-hour hackathon and it may take time for your hardware to be delivered.

Comment out sponsors

Get rid of "Check out last year" section

Refer to figma: https://www.figma.com/file/hVWlYzcRfwFgk17LFUqw8Z/NewHacks-%2B-MakeUofT-website-redesign?node-id=421%3A388

### CODE REVIEW

**Comment by Leo6Leo** [comment]
Link to [Jira ticket](https://ieeeuoft.atlassian.net/browse/IEEE-57?atlOrigin=eyJpIjoiNTZmNmYyMmY5NGY0NGZlNzk1ZjhmMGNmYjJlOGZlYjEiLCJwIjoiaiJ9)

File: hackathon_site/event/jinja2/event/landing.html [hunk 1]
```diff
@@ -207,23 +207,21 @@ <h3 class="center white-text">FAQ</h3>
...
                         <li>
                             <div class="collapsible-header">
-                                How do online hackathons work?
+                                Is this hackathon online?
                             </div>
                             <div class="collapsible-body">
                                 <span>
-                                    The entire hackathon will be conducted online. You will be able to stay safe and comfortably at home while doing this hackathon. You will be able to communicate to your team via
-                                    online meetings and you will be submitting your final products online.
+                                    Yes, this hackathon will be hosted completely online using multiple online communication platforms. So you can enjoy the hacking experience from the comfort of your home.
```
Comment (lisa-sa-li) [inline hackathon_site/event/jinja2/event/landing.html:224]: Minor gramatical change: 
```suggestion
Yes, this hackathon will be hosted completely online using multiple online communication platforms so you can enjoy the hacking experience from the comfort of your home.
```

File: hackathon_site/event/jinja2/event/landing.html [hunk 2]
```diff
@@ -234,44 +232,37 @@ <h3 class="center white-text">FAQ</h3>
...
-            <div class="row sponsorImgs">
-                <!-- Platinum -->
-                <a href="https://www.daisyintelligence.com/" class="col m6 s12" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Daisy_Intelligence_white.svg') }}" alt="Daisy Intelligence"/></a>
-                <a href="https://tractionondemand.com/" class="col m6 s12" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Traction_On_Demand_white.svg') }}" alt="Traction on Demand" /></a>
+<!--<div class="ombreBackgroundDark2">-->
+<!--    <div class="container section scrollspy" id="sponsors">-->
+<!--        <div class="row">-->
+<!--            <h3 class="center white-text">Sponsors</h3>-->
+<!--            <div class="row sponsorImgs">-->
+<!--                &lt;!&ndash; Platinum &ndash;&gt;-->
```
Comment (lisa-sa-li) [inline hackathon_site/event/jinja2/event/landing.html:240]: Undo the changes on the already-commented lines. No need to render `&lt;!&ndash;`, just keep the line as is because it's already a comment. Same for the lines below: Gold, Silver, Bronze, Partners

**Review by lisa-sa-li — COMMENTED** [review]
We need to run a linter on the file cause the tabs are a bit off.

@grahamhoyes do you remember how we formatted the html files? Prettier has a hard time reading jinja templates.

**Comment by lisa-sa-li** [comment]
Also really minor, can you change the name of the PR to be more descriptive and contain the PR #, e.g.:

IEEE-57: update hackathon information on landing page

**Comment by Leo6Leo** [comment]
> Also really minor, can you change the name of the PR to be more descriptive and contain the PR #, e.g.:
> 
> IEEE-57: update hackathon information on landing page

File: hackathon_site/event/jinja2/event/landing.html (hunk 2 above)
Comment (Leo6Leo) [reply hackathon_site/event/jinja2/event/landing.html]: I see. At first, I just select all and ctrl+/ to comment out the sponsor part.

**Comment by grahamhoyes** [comment]
> We need to run a linter on the file cause the tabs are a bit off.
> 
> @grahamhoyes do you remember how we formatted the html files? Prettier has a hard time reading jinja templates.

Unfortunately, we don't have a linter for Jinja templates. We just have to take care of them ourselves.

File: hackathon_site/event/jinja2/event/landing.html [hunk 3]
```diff
@@ -181,11 +181,11 @@ <h3 class="center white-text">FAQ</h3>
                         </li>
                         <li>
                             <div class="collapsible-header">
-                                When can I expect to hear back after I applied?
+                                Can we buy hardware and use it in our project?
                             </div>
                             <div class="collapsible-body">
                                 <span>
-                                    There will be 1 round of acceptances tentatively scheduled for Oct 26th/27th. We review applications on a rolling basis so apply early!
+Yes, you may buy hardware and use it towards your project. However, please remember that this is only a 24-hour hackathon and it may take time for your hardware to be delivered.
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:188]: Indentation fix

File: hackathon_site/event/jinja2/event/landing.html [hunk 4]
```diff
@@ -207,23 +207,21 @@ <h3 class="center white-text">FAQ</h3>
...
                         <li>
                             <div class="collapsible-header">
-                                What equipment and support will be provided?
+                                What software/hardware can we use?
                             </div>
                             <div class="collapsible-body">
                                 <span>
-                                    Unfortunately, due to the current situation of COVID19, we won’t be able to provide any hardware. However, there will be lots of mentors ready to help you! During the online
-                                    hackathon, there will be a breakout room for all the mentors where you will be able to access at all times and get immediate help.
+                                As this hackathon is online, you can use any of the software or hardware that you have available at your house.
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:214]: Indentation fix

File: hackathon_site/event/jinja2/event/landing.html [hunk 5]
```diff
@@ -207,23 +207,21 @@ <h3 class="center white-text">FAQ</h3>
...
                         <li>
                             <div class="collapsible-header">
-                                How do online hackathons work?
+                                Is this hackathon online?
                             </div>
                             <div class="collapsible-body">
                                 <span>
-                                    The entire hackathon will be conducted online. You will be able to stay safe and comfortably at home while doing this hackathon. You will be able to communicate to your team via
-                                    online meetings and you will be submitting your final products online.
+Yes, this hackathon will be hosted completely online using multiple online communication platforms so you can enjoy the hacking experience from the comfort of your home.
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:224]: Indendation fix

**Review by grahamhoyes — COMMENTED** [review]
Just the indentation stuff, other than that all good

**Comment by grahamhoyes** [comment]
Logistics also asked if we can change  "24 Hour Beginner Friendly Digital Hackathon" to "24 Hour Beginner Friendly Online Hackathon", can you make that quick change?

**Review by grahamhoyes — APPROVED** [review]
Good stuff!
